### PR TITLE
PAY-ADMIN - Fix for issues with cookies, also fix OIDC so it uses JWKS etc.

### PR DESCRIPTION
**Pull request by seeker25**
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-pay license (Apache 2.0).

### CODE REVIEW

File: pay-admin/admin/config.py [hunk 1]
```diff
@@ -86,12 +87,18 @@ class _Config():  # pylint: disable=too-few-public-methods
 
     # Normal Keycloak parameters.
     OIDC_CLIENT_SECRETS = os.getenv('PAY_OIDC_CLIENT_SECRETS', 'secrets/keycloak.json')
+    with open(OIDC_CLIENT_SECRETS, 'r', encoding='utf-8') as file:
+        secrets = json.load(file)
+    JWT_OIDC_JWKS_URI = secrets['web']['jwks_uri']
```
Comment (seeker25) [inline pay-admin/admin/config.py:92]: I've added these strings to DEV/TEST/PROD

File: pay-admin/admin/config.py [hunk 2]
```diff
@@ -92,6 +93,7 @@ class _Config():  # pylint: disable=too-few-public-methods
     OIDC_ID_TOKEN_COOKIE_SECURE = os.getenv('PAY_OIDC_ID_TOKEN_COOKIE_SECURE', 'True').lower() == 'true'
 
     PREFERRED_URL_SCHEME = 'https'
+    SESSION_TYPE = 'filesystem'
```
Comment (seeker25) [inline pay-admin/admin/config.py:96]: Uses smaller cookie, saves 250-500 of these on the drive

File: pay-admin/poetry.lock [hunk 3]
```diff
@@ -811,6 +811,30 @@ files = [
 [package.dependencies]
 Flask = "*"
 
+[[package]]
+name = "flask-session"
```
Comment (seeker25) [inline pay-admin/poetry.lock:815]: This library makes the cookie very small, saves most of the session on the disk

**Review by ochiu — APPROVED** [review]
Looks good. Thanks for the comments explaining the changes.

**Pull request merged**